source/module_pw/test/test1-2-2.cpp

```diff
@@ -0,0 +1,158 @@
+//---------------------------------------------
+// TEST for FFT
+//---------------------------------------------
+#include "../pw_basis.h"
+#ifdef __MPI
+#include "test_tool.h"
+#include "../../src_parallel/parallel_global.h"
+#include "mpi.h"
+#endif
+#include "../../module_base/constants.h"
+#include "../../module_base/global_function.h"
+#include "pw_test.h"
+
+using namespace std;
+TEST_F(PWTEST,test1_2_2)
```

Comment:
What's the meaning of `test1_2_2` here?

Reply:
different cases for pw_module
information is "couted"
cout<<"dividemthd 1, gamma_only: off, double precision, check fft between double and complex"<<endl;